module-4/backend/week_four.md

```diff
@@ -5,15 +5,38 @@ Fork this respository. Answer the questions to the best of your ability. Try to
+To simulate live conditions without the repercusions of live conditions.  For the same reason flight schools use bad weather simulators instead of actually sending flight students to their possible death in adverse weather conditions (this is actually the hardest part of flight school and where most people wash out, my school had a 10% pass rate!).
+
 3. What are the characteristics of a good README (in your opinion)?
+Based on my QS assessment I think I am the wrong person to ask but what I look for in a READ ME for a new gem is 1. easy to understand, 2. assume nothing of the reader, spell things out, and have the most useful information at the top.  This is the same for good writing in general but for techincal writing it is extra important.  Something I should have done on QS in retrospect was to follow our installation instructions to see if they were easy to follow and if I missed any steps.  Oops.
+
 4. What's one main improvement you're going to make to your code regarding accessibility issues?
+For QS we intentionally used buttons on everything for ease of accessability.  That and contrasting colors for our app.  After using Vim for three months I want to go full keyboard, even in Chrome, so I hope this will really help me understand the importance of accessability because I too won't be using a mouse.  
+
 5. What are some basic security concerns to be aware of when building applications?
+The most intersting one to me is clickJacking, or where hidden buttons do hidden actions to unsuspecting users.  For me, I think being aware of the problem and getting some time using some of the gems out there to test vulnerabilities in your code is a good idea.  I talked to a Turing grad over the break for job advice and he told me that their company got hacked and it was his code that added the vulnerability.  It was very sobering to hear.  This is important stuff.  I know when I worked with the Senate Intel Committee on Cybersecurity issues this was something that came up alot.  My Senate office was constantly targeted by foreign governments and as our Sys Admin I renwed my Secret clearance so I could better defend us from attacks.  Even my secret clearance info (along with millions of others) was stolen by hackers a year or two ago.  Much, much worse for the US Gov than Equifax.  This is important stuff and it affects everyone.  
+
 6. Why is continuous integration helpful/important?
+Continuous integration (CI) is the practice of merging all developer working copies to a shared mainline several times a day (like github) and testing every build to make sure it works (like with TravisCI).  For me, as programs get bigger and you are reviewing lots of code, the first question you ask is "does this code even work?".  CI products solve this problem by running the code for you to make sure it works.  When I was jobshadowing with Healthify I asked them about this and they used Travis CI (and it wasn't able to run the app while I was shadowing!).  
```

Comment:
great response here! and using a CI tool allows you to work on something else while the entire test suite runs.. which is important too!